### DISCUSSION TITLE
Copy to clipboard on a custom link

### DISCUSSION THREAD

**Discussion by AliMickey** [netbox-community/netbox#18178] [Help Wanted!]
Hi,

I have a custom link that contains the following logic: 
`ssh user@{{object.primary_ip4.address.ip}}`

At the moment this results in the link being: 
`https://dcim.domain.com/dcim/devices/id/ssh%20user@primary_ip` 

Which is expected somewhat as the purpose of custom links is to inherently being a link. 

I would like an option so that the calculated 'link' is copied to clipboard when the link button is clicked on. 

I have tried doing 'ssh://' which does work, however does not fit my other use cases with custom ssh parameters (`ssh -i` eg).

Are there any suggestions on how this can be achieved?

**Comment by PieterL75** [comment]
you can create your own handler 'mycoolssh://' and have that linked to a bash/bat/ps/py script that handles the link

**Reply by AliMickey** [reply]
Thanks for the workaround, unfortunately not scalable with many users and would prefer native functionality.
